ait/gui/__init__.py

```diff
@@ -127,21 +127,31 @@ class Playback(object):
     A Playback manages the state for the playback component.
     playback.dbconn: connection to database
     playback.query: time query map of {timestamp: list of (uid, data)} from database
-    playback.on: true if gui is currently in playback mode
+    playback.on: True if gui is currently in playback mode. Real-time telemetry will not
+        be sent to the frontend during this.
```

Comment:
Just curious is stopping real-time telem the intended functionality, or something to change in the future?

Reply:
That's intended. If we're playing back telemetry from a historical query we cut off the real time stream to the clients so the older data can be replayed.